pages/home.html

```diff
@@ -21,20 +27,18 @@
     Bite-sized
   </h3>
   <p>
-    Explorations are easy to create and quick to play. They can be published in
-    the exploration gallery, shared on social media, or embedded on your
-    webpage.
+    Easy to create and quick to play, explorations can also be published in the exploration gallery, shared on social media, or embedded on your webpage.
```

Comment:
But the resulting sentence is long and combines lots of ideas. I think it feels a bit clunky.

(Speaking of understandability -- did you user test? It might be worth doing that with some of your friends, or folks on the gitter chat.)